src/scripts/clipperUI/componentBase.ts

```diff
@@ -67,89 +75,164 @@ export abstract class ComponentBase<TState, TProps> {
 	}
 
 	/*
-	 * Helper which handles tabIndex, clicks, and keyboard navigation.
-	 *
-	 * Also hides the outline if they are using a mouse, but shows it if they are using the keyboard
-	 * (idea from http://www.paciellogroup.com/blog/2012/04/how-to-remove-css-outlines-in-an-accessible-manner/)
-	 *
-	 * Example use:
-	 *      <a id="myCoolButton" {...this.enableInvoke(this.myButtonHandler, 0)}>Click Me</a>
-	 */
-	public enableInvoke({callback = undefined, tabIndex = 0, args = undefined, idOverride = undefined, setNameForArrowKeyNav = undefined}: EnableInvokeParams) {
-		// Because of the way mithril does the callbacks, we need to rescope it so that "this" points to the class
+		 * Helper which handles tabIndex, clicks, and keyboard navigation for a component that is part of an Aria Set
+		 *
+		 * Also hides the outline if they are using a mouse, but shows it if they are using the keyboard
+		 * (idea from http://www.paciellogroup.com/blog/2012/04/how-to-remove-css-outlines-in-an-accessible-manner/)
+		 *
+		 * Example use:
+		 *      <a id="myCoolButton" {...this.enableInvoke(this.myButtonHandler, 0)}>Click Me</a>
+		 */
+	enableAriaInvoke({callback = undefined, tabIndex = 0, args = undefined, idOverride = undefined, ariaSetName = undefined, ariaSetDirection = undefined}: EnableAria) {
```

Comment:
Some of these params should be required, not optional. For public methods/functions, it's good practice to check all required inputs to make sure the right thing is being passed in.